Makefile

```diff
@@ -56,24 +56,29 @@ lint:
 	rm requirements.commit
 
 functional-tests-exitfirst:
-	pytest --exitfirst --gherkin-terminal-reporter tests -vvv --cucumberjson=test-report.json --durations=0 --html=test-report.html
+ifeq ($(SLOW),)
+	echo "Running fast tests"
+	pytest --exitfirst --splits 3 --group $(PYTEST_GROUP) --store-durations --durations-path=./.test_durations.$(PYTEST_GROUP) --deselect=tests/functional/targets/test_performance.py::test_api_requester_multiple_request_and_targets[50-50-600] tests -vvv --cucumberjson=test-report.json --durations=0 --html=test-report.html
+else
+	echo "Running slow tests"
+	pytest --exitfirst tests/functional/targets/test_performance.py::test_api_requester_multiple_request_and_targets[50-50-600] -vvv --cucumberjson=test-report.json --durations=0 --html=test-report.html
+endif
 
 functional-tests:
 	pytest --gherkin-terminal-reporter tests -vvv --cucumberjson=test-report.json --durations=0 --html=test-report.html
 
-
 # CLI_VERSION enables using specific RSTUF CLI version, default: dev
 # usage: `make ft-das CLI_VERSION=v0.8.0b1`
 ft-das:
 ifneq ($(CLI_VERSION),)
-	docker compose run --env UMBRELLA_PATH=. --rm rstuf-ft-runner bash tests/functional/scripts/run-ft-das.sh dev
+	docker compose run --env UMBRELLA_PATH=. --rm rstuf-ft-runner bash tests/functional/scripts/run-ft-das.sh dev $(PYTEST_GROUP) $(SLOW)
```

Comment:
How does docker-compose use `PYTEST_GROUP` and `SLOW`?

Reply:
docker-compose is starting the [`rstuf-ft-runner` service ](https://github.com/repository-service-tuf/repository-service-tuf/blob/a5f92f092ec22d8e4cc8b6dda1925ca22755357b/docker-compose.yml#L76) and runs the command `bash tests/functional/scripts/run-ft-das.sh dev $(PYTEST_GROUP) $(SLOW)`.
`PYTEST_GROUP` and `SLOW` are passed as parameters to the `tests/functional/scripts/run-ft-das.sh` bash script and are later on used to:
- define the testing group for `pytest-split` 
- run the slowest by far test in a separate job

respectively.